securedrop_client/logic.py

```diff
@@ -47,33 +47,29 @@ def __init__(self, api_call, *args, **kwargs):
         self.args = args
         self.kwargs = kwargs
         self.result = None
+        self.i_timed_out = False
```

Comment:
Flag telling the API thread that it's been timed out, and when it finally gets unblocked it should Do Nothing.